### PR TITLE
LRQA-54229 | Master

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LRQA-54229
AppBuilder#PlaceFormViewWidget

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hey @nkkwon , could you please take a look again?

**Comment by nkkwon** [comment]
Issues should be fixed @alexcandidos

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 1]
```diff
@@ -1403,6 +1403,54 @@ definition {
...
+		var myAppName = "My Registration app";
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Text");
+		FormViewBuilder.setFieldLabel(fieldLabel = "Full name");
+		FormViewBuilder.setFieldRequired();
```
Comment (alexcandidos) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Hi @nkkwon, you don't need to set this field as required if you are not going to validate it.

**Comment by alexcandidos** [comment]
Pull request forwarded to https://github.com/brianchandotcom/liferay-portal/pull/84243. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alexcandidos** [comment]
Thanks @nkkwon !

**Pull request closed without merging**